src/hotspot/share/oops/array.hpp

```diff
@@ -74,12 +74,12 @@ class Array: public MetaspaceObj {
     assert(is_aligned(left, sizeof(T)), "Must be");
 
     size_t elements = left / sizeof(T);
-    assert(elements <= (size_t)INT_MAX, "number of elements " SIZE_FORMAT "doesn't fit into an int.", elements);
+    assert(elements <= (size_t)INT_MAX, "number of elements %zudoesn't fit into an int.", elements);
```

Comment:
Nit: It seems that a space is needed after `%zu`. This was this way originally.

Reply:
I added the space, since you read through carefully enough to see it.  Thanks!